api_spec.md

```diff
@@ -374,7 +367,49 @@ type FilterDistanceConfig = {
     // since it will not update the filter/overlay until polyline moves/edits are complete.
 }
 ```
-Where all `config_data` properties are optional.
+
+### `change_zoom_keybind`
+Keybind to change the zoom level. Must be a letter, and the lowercase version of the letter will set the zoom level to the `initial_crop`, while the capitalized version will show the full image. Default is `r`.
```

Comment:
Do we need a manually-written section for all of these, or can these be JSDoc/TSDoc that then generates a portion of this document?
If we want to go this route, we don't need to decide on something now - I can open an issue for it.

Reply:
Yeah we can def move JSDoc in the code that generates this, that will probably have the best shot of actually staying up to date

Reply:
yeah we punt 🏈

Reply:
Opened #172 to track this.